vmdb/spec/requests/api/vms_spec.rb

```diff
@@ -637,4 +643,72 @@ def gen_request_data(action, data, *hrefs)
       expect(vm2.reload.evm_owner).to eq(@user)
     end
   end
+
+  context "Vm add_event action" do
+    it "add_event to an invalid vm" do
```

Comment:
Saying "add_event" to the it here is redundant.  RSpec builds a sentence, so this is "ApiController Vm add_event action add_event to an invalid vm"